### ISSUE TITLE
Could support vue3?

### ISSUE THREAD

**Issue by ruanjianbai** [YishaiBerg/vue-dynamic-marquee#18]
Hi, I use this in my vue3 Project, but show error:' Cannot read property 'extend' of undefined', How should I do? Thank you!

**Comment by YishaiBerg** [comment]
Yeah, I certainly do plan to rewrite the component in Vue 3, but between my job and family I find it a bit hard to make time for it. Hopefully I will be able to get around to it during the next month or two. If anyone will like to submit a PR for this they are much welcome to do so, just post here that you are working on it.

**Comment by tintin10q** [comment]
Do you think you will have time any time soon?

**Comment by YishaiBerg** [comment]
I really want to get around to it, but I have been super busy this past year. I hope I will find time for it over the next months, but I can't promise.
If anyone wants to work on it (shouldn't be too hard IMO) I will make time to review the PR, and give credit in the README.
